src/main/kotlin/org/consoletrader/candles/bitmarket/BitmarketPublicApi.kt

```diff
@@ -0,0 +1,10 @@
+package org.consoletrader.candles.bitmarket
+
+import io.reactivex.Observable
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface BitmarketPublicApi {
+    @GET("/json/{pair}/ticker.json")
```

Comment:
It's a wrong endpoint. It's only ticker (no historic data) price with no time. We need to use this: https://www.bitmarket.pl/graphs/BTCPLN/1d.json
It's the closest we can get (returning candles every 16 minutes)
We also need to skip every second candle to be as close to 30 minutes as possible (other exchanges data).